src/PayloadListener/Services/IEventPayloadRecieverService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Monai.Deploy.Messaging.Common;
+
+namespace Monai.Deploy.WorkflowManager.PayloadListener.Services
```

Comment:
@jackschofield23 shouldn't be abstract class be agnostic as to which event has been received?

Reply:
Not quite sure what you mean here, but I was thinking that we can define a separate callback method to be called for each queue, which will then each call a different method in the EventPayloadRecieverService to decide how to handle the payload.

Reply:
Got you, so this interface is generic and will include abstract methods for each queue, right?

Reply:
This isn't currently set up as an abstract class, I was planning to have a different method for each payload type, that will be called from the payload listener service in each relevant callback.